tenacity/__init__.py

```diff
@@ -56,7 +54,12 @@
 from .after import after_log  # noqa
 from .after import after_nothing  # noqa
 
+from .marshaller import Future  # noqa
```

Comment:
Actually, it looks like it's used only in tests. But just to *not touch tests* at all in this PR, I added this imports.